### ISSUE TITLE
Population time profiles: Legend adjustments

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite.ReportingEngine#463]
1. Include the name of the simulation set also for non-reference pop.
Thus for the example below it should be `Simulated mean for Raltegravir (Larson 2013 8y-18y 400mg FCT meal)` etc. 
2. Insert `Observed data` before the observed data name
Thus for the example below it should be `Observed data Observed_Raltegravir 10 mg (lactose formulation)` etc. 
```
simSet1 <- PopulationSimulationSet$new(simulationSetName = 'Larson 2013 8y-18y 400mg FCT meal', 
                                       simulationFile = file.path(workflowDir, "Models", "Larson 2013 8-18y meal.pkml"),
                                       populationFile = file.path(workflowDir, "Models", "Larson 2013 8-18y meal-Population.csv"),
                                       populationName = "Pediatric population 8y-18y", 
                                       referencePopulation = FALSE,
                                       studyDesignFile = NULL,
                                       outputs = outputVenousBlood('Grouping=="8-18y"', 'Observed_Raltegravir 10 mg (lactose formulation)',  pkVenousSingle), 
                                       observedDataFile = dataFile, observedMetaDataFile = dictionaryFile)

simSet2 <- PopulationSimulationSet$new(simulationSetName = 'Filmcoated_tablet_400mg_sd', 
                                       simulationFile = file.path(workflowDir, "Models", "Raltegravir 400mg filmcoated tablet.pkml"),
                                       populationFile = file.path(workflowDir, "Models", "Raltegravir Adult Population.csv"),
                                       populationName = "Healthy Adult Population", 
                                       referencePopulation = TRUE,
                                       studyDesignFile = NULL,
                                       outputs = outputVenousBlood('Grouping=="400mg_FCT"', 'Observed_Raltegravir 400mg filmcoated tablet',  pkVenousSingle), 
                                       observedDataFile = dataFile, observedMetaDataFile = dictionaryFile)
```
![grafik](https://user-images.githubusercontent.com/25061876/114881817-a24d5680-9e03-11eb-897b-2d34db05b900.png)

**Comment by pchelle** [comment]
@Yuri05 
- If the user set no reference population (e.g. parallel comparison), do we still add the simulation set name anyways ?
- Same question for mean model workflows ?
- Do we also add the terms "Simulated" and "Observed data" before the output display for mean model workflows ?

**Comment by Yuri05** [comment]
@pchelle YES to all questions. Good that you noticed that :)

**Comment by Yuri05** [comment]
Fixed with #466
